webapp/index.html

```diff
@@ -34,7 +34,7 @@
 
 <header class="side">
     <a class="artist-link" href="https://justinholmes.com/">
-        <p class="about-band">Home</p>
+        <p class="about-band"><img src="images/thumbnails/jh-logo-shaded-white.svg"></p>
```

Comment:
![image](https://github.com/jMyles/VowelSoundsAlbumDrop/assets/12272640/d303b8ce-d14e-4e64-8e5b-c5e57a58486c)
The image doesn't load in brave @rjpartingtoniii